### PR TITLE
RA-1625: Unable to display inactive conditions

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1625:Unable to display inactive conditions

## Description of what I changed
show inactive conditions

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1625

## Checklist: I completed these to help reviewers :)

- [X] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [X] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [X] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [X] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [X] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [X] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you see the travis failure?

File: web-2.2/src/main/java/org/openmrs/module/emrapi/web/controller/ConditionController.java [hunk 1]
```diff
@@ -374,9 +374,9 @@ private ConditionClinicalStatus convertConditionListStatus(Condition.Status stat
 	@RequestMapping(method = RequestMethod.GET, value = "/conditionhistory")
 	@ResponseBody
 	public List<org.openmrs.module.emrapi.conditionslist.contract.ConditionHistory> getConditionHistory(@RequestParam("patientUuid") String patientUuid) {
-		List<org.openmrs.Condition> activeConditions = conditionService.getActiveConditions(patientService.getPatientByUuid(patientUuid));
+		List<org.openmrs.Condition> conditions = conditionService.getAllConditions(patientService.getPatientByUuid(patientUuid));
```
Comment (mozzy11) [inline web-2.2/src/main/java/org/openmrs/module/emrapi/web/controller/ConditionController.java]: @haripriya999   , Obviously this method doesnt exist any where in the core, it still lives in your PR , theres no way you can call it from here . Until your Pr is merged and  at the same time you depend on the latest changes in the master , or probably its back ported to the core version that core apps depends on

Comment (haripriya999) [reply web-2.2/src/main/java/org/openmrs/module/emrapi/web/controller/ConditionController.java]: @mozzy11  Thank you for your insight on this. Is it that every time the module build takes places, it uses internet to get the latest changes?

**Comment by haripriya999** [comment]
> Did you see the travis failure?

@dkayiwa  There is a travis error as it is not able to find getAllConditions fucntions that I defined in core: https://github.com/openmrs/openmrs-core/pull/2996 . As @mozzy11  described we will have to get this PR: https://github.com/openmrs/openmrs-core/pull/2996 merged to fix the travis error.  :)

**Comment by dkayiwa** [comment]
@haripriya999 are you still working on this?

**Comment by haripriya999** [comment]
@dkayiwa  This PR is ready to be merged :)

**Comment by haripriya999** [comment]
@dkayiwa I'm not sure why I'm seeing this error: https://pastebin.com/jq92mCnR , in travis logs though the implementation of getAllConditions() in core PR:https://github.com/openmrs/openmrs-core/pull/2996  has been merged.

**Comment by dkayiwa** [comment]
@haripriya999 did you see the merge conflicts?

**Comment by mozzy11** [comment]
@dkayiwa , unfortunately i just did this same resolution and it was merged
here https://github.com/openmrs/openmrs-module-emrapi/pull/176 . 
Unfortunately i hadnt known the @haripriya999 had also tried to do the same thing earler.
I think @haripriya999  ,didnt report this PR on the ticket here https://issues.openmrs.org/browse/RA-1

**Comment by mozzy11** [comment]
@haripriya999  , However , Even though i fixed this , 
 we still need some more slight fix in the Angular Client component of the Condition List in Core apps Even though now the request fetches all the conditons as expected,  displaying inactive conditions is still a little  buggy. See Ellen's comment here https://issues.openmrs.org/browse/RA-1641

**Pull request closed without merging**